src/main/java/com/exasol/adapter/sql/UserInformation.java

```diff
@@ -3,17 +3,74 @@
 import com.exasol.adapter.jdbc.RemoteMetadataReaderException;
 
 import java.sql.*;
+import java.util.logging.Logger;
 
+/**
+ * This class collect information about user's roles.
+ */
 public class UserInformation {
+    private static final Logger LOGGER = Logger.getLogger(UserInformation.class.getName());
+    private static final int MAX_ROLE_VALUE = 64;
```

Comment:
Actually the maximum value is 2^62.
Don't forget, every bit represents a role the user has.
The highest bit is reserved for public access.

Reply:
fixed